sdk/python/kfp/_client.py

```diff
@@ -115,6 +115,8 @@ class Client(object):
   IN_CLUSTER_DNS_NAME = 'ml-pipeline.{}.svc.cluster.local:8888'
   KUBE_PROXY_PATH = 'api/v1/namespaces/{}/services/ml-pipeline:http/proxy/'
 
+  NAMESPACE_PATH = '/var/run/secrets/kubernetes.io/serviceaccount/namespace'
```

Comment:
Can you add a comment linking to documentation saying this file is auto populated in kubernetes pods?